frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/vcsmenu/OpenMacheteTabAction.java

```diff
@@ -23,17 +25,30 @@ public IEnhancedLambdaLogger log() {
   public void actionPerformed(AnActionEvent e) {
     LOG.debug("Performing");
 
+    // Getting project from event and assigning it to variable is needed to avoid exception
```

Comment:
Exception or rather just a race condition (possibly not causing any expections)?

Reply:
Unfortunately it caused exception

Reply:
Ok so for clarity pls include what exception specifically it was if you remember

Reply:
I was just `Throwable` with proper message